Update iOS plugins documentation (#4693)
godotengine/godot-docs: tutorials/platform/ios/plugins_for_ios.rst

.. _doc_plugins_for_ios:

Plugins for iOS
===============

At the moment Godot provides StoreKit, GameCenter, iCloud services plugins.
They are using same model of asynchronous calls explained below.

ARKit and Camera access are also provided as plugins.

Accessing plugin singletons
---------------------------

> To access plugin functionality it's first required to use check if plugin
> is exported and available with `Engine.has_singleton` function. After
> that calling a `Engine.get_singleton` will return a singleton. This
> is an example of how this can be done:

Reviewer: ```suggestion
To access plugin functionality, you first need to check that the plugin is
exported and available by calling the `Engine.has_singleton()` function, which
returns a registered singleton.

Here's an example of how to do this in GDScript:
```

::

    var in_app_store

    func _ready():
        if Engine.has_singleton("InAppStore"):
            in_app_store = Engine.get_singleton("InAppStore")
            

>             # Plugin setup
>         else:
>             print("iOS IAP plugin is not exported.")

Reviewer: ```suggestion
        else:
            print("iOS IAP plugin is not exported.")
```
